### PR TITLE
Update AZ-203_03_lab_ak.md

### PR DESCRIPTION
**Pull request by cdanvergara**
Added more accurate steps to create the Azure SQL Database, the flow was difficult to follow creating first the server and adding a suggested name for the database.

## Location

- Course: 00
- Exercise: 00
- Task: 00
- Step[s]: 00

# Proposed Changes

-
-
-

### CODE REVIEW

File: Instructions/Labs/AZ-203_03_lab_ak.md [hunk 1]
```diff
@@ -120,37 +120,35 @@ Observe the taskbar located at the bottom of your **Windows 10** desktop. The ta
...
-    > **Note**: Each tab represents a step in the workflow to create a new **Azure SQL Database server**. At any time, you can select **Review + create** to skip the remaining tabs.
+1.  In the **Create SQL Database** blade, view your list of SQL server instances.
 
 1.  In the **Basics** tab, perform the following actions:
 
     1.  Leave the **Subscription** drop-down list set to its default value.
 
     1.  In the **Resource group** section, select **PolyglotData** from the list.
+    
+    1.  In the **Database name** field, enter the value **polysqldb\[your name in lower case\]**.
```
Comment (seesharprun) [inline Instructions/Labs/AZ-203_03_lab_ak.md]: Why are you naming this ``polysqldb``? If your intention is to build a database you would never use, then do you want a name to indicate that this is being discarded?

File: Instructions/Labs/AZ-203_03_lab_ak.md [hunk 2]
```diff
@@ -120,37 +120,35 @@ Observe the taskbar located at the bottom of your **Windows 10** desktop. The ta
 
 1.  In the **Access keys** pane, record the value in the **Primary connection string (StackExchange.Redis)** field. You will use this value later in this lab.
 
-#### Task 3: Create an Azure SQL server resource
+#### Task 3: Create an Azure SQL Database
 
 1.  In the navigation pane on the left side of the Azure portal, select **All services**.
 
-1.  In the **All services** blade, select **SQL servers**.
+1.  In the **All services** blade, select **SQL Database**.
```
Comment (seesharprun) [inline Instructions/Labs/AZ-203_03_lab_ak.md]: Did you tell the students to click the **+ Add** button to add a new database?

File: Instructions/Labs/AZ-203_03_lab_ak.md [hunk 3]
```diff
@@ -120,37 +120,35 @@ Observe the taskbar located at the bottom of your **Windows 10** desktop. The ta
...
 
     1.  In the **Password** field, enter the value **TestPa$$w0rd**.
 
     1.  In the **Confirm password** field, enter the value **TestPa$$w0rd** again.
-
-    1.  Select **Next: Networking >**.
+    
+    1.  In the **Location** drop-down list, select the **(US) East US** option and click **OK**.
+    
+    1. Leave the other values by default and select **Next: Networking >**.
```
Comment (seesharprun) [inline Instructions/Labs/AZ-203_03_lab_ak.md]: We need to specify the database size here.

**Comment by seesharprun** [comment]
I don't see the value in adding this PR and am closing the issue as not accepted.

If you disagree, can you elaborate on the following points and re-open the issue:

- *"the flow was difficult to follow creating first the server and adding a suggested name for the database"* 
  - > The students are only creating a server at this point, not a database. We don't need a suggested name for the database because the students are creating a database yet.
- *"Added more accurate steps to create the Azure SQL Database"*
  - > Which steps were incorrect? I had my tester look at the lab instructions and generate a screenshot validating that they are accurate to create an Azure SQL server, not a database.
    image

File: Instructions/Labs/AZ-203_03_lab_ak.md (hunk 1 above)
Comment (cdanvergara) [reply Instructions/Labs/AZ-203_03_lab_ak.md]: Hi Andrew, nevermind I see that you are just creating the server and later on importing the database using the bacpac file.

Comment (seesharprun) [reply Instructions/Labs/AZ-203_03_lab_ak.md]: No worries, I like that people are getting involved in the repo.

**Pull request closed without merging**